README.md

```diff
@@ -121,6 +122,7 @@ Solomon proposed to support VXLAN from the Docker core.
 - Flynn
 - [Modit](https://mod.it): for staging.
 - Shippable: for testing & deployment
+- Vagrant 
```

Comment:
To be more specific, like `[Vagrant's Docker provisioning](http://docs.vagrantup.com/v2/provisioning/docker.html)`?